app/src/styles/Report.module.css

```diff
@@ -0,0 +1,117 @@
+.reportForm {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    background-color: #fdf8f5;
+    color: #4f4846;
+    width: 100%;
+    padding: 20px;
+    border-radius: 5px;
+    box-shadow: 0px 0px 10px rgba(0, 0, 0, 0.15);
+    position: relative;
+}
+.formColumn {
```

Comment:
Non-blocking: It's conventional for CSS to include an empty line between different statements, and it's conventional to use `.lower-kebab-case` rather than `lowerCamelCase`